airbyte-webapp/src/locales/en.json

```diff
@@ -471,7 +471,8 @@
   "connector.connectorsInDevelopment.beta": "<b>Beta connectors</b> are in development but stable and reliable and support is provided. See our <lnk>documentation</lnk> for more details.",
   "credits.credits": "Credits",
   "credits.whatAreCredits": "What are credits?",
-  "credits.buyCredits": "+ Buy credits",
+  "credits.buyCredits": "Buy credits",
```

Comment:
hmm, shouldn't it be in cloud locales?

Reply:
Indeed it most likely should. But since all the credit strings are currently in this file I'd leave this change for now here, and rather pull moving them all over into a separate PR.